Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1084,6 +1085,18 @@ eval (MAPREADY key latitude longitude) state =
         _ <- checkPermissionAndUpdatePersonMarker state
         pure AfterRender
       ]
+  where 
+    callLocateOnMap state = do
+        pure $ unsafePerformEffect $ runEffectFn1 locateOnMap locateOnMapConfig 
+            {   lat = state.props.sourceLat
+              , lon = state.props.sourceLong
+              , geoJson = state.data.polygonCoordinates
+              , points = state.data.nearByPickUpPoints
+              , labelId = getNewIDWithTag "LocateOnMapPin"
+              , zoomLevel = zoomLevel
+              , locationName = fromMaybe "" state.props.locateOnMapProps.sourceLocationName
+              , specialZoneMarkerConfig { labelImage = zoneLabelIcon state.props.confirmLocationCategory, labelMaxWidth = locateOnMapLabelMaxWidth} 
```

Comment:
`zoomLevel` and `labelMaxWidth` as they are same as default value

Reply:
not related to this but can you add default value to `labelMaxWidth` in `locateOnMapConfig` from `Constant`

Reply:
fixed